clients/python/test/e2e/test_solvers_api.py

```diff
@@ -50,18 +33,19 @@ def test_get_jobs(configuration: osparc.Configuration):
             )
             created_job_ids.append(job.id)
 
-        tmp_iter = solvers_api.get_jobs(
-            sleeper.id, sleeper.version, limit=3, offset=n_init_iter
-        )
+        tmp_iter = solvers_api.list_jobs(sleeper.id, sleeper.version)
+        solvers_api.list_jobs(sleeper.id, sleeper.version)
 
-        final_iter = solvers_api.get_jobs(sleeper.id, sleeper.version, limit=3)
+        final_iter = solvers_api.list_jobs(sleeper.id, sleeper.version)
         assert len(final_iter) > 0, "No jobs were available"
```

Comment:
`final_iter` is a `PaginationGenerator`. I decided to create that intermediary class instead of directly returning an iterator in order for it to have a `__len__` method. However, `PaginationGenerator` has an `__iter__` method, so 
```python
for elm in final_iter:
    ...
```
works